Index: app/models.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/models.py	(revision )
+++ app/models.py	(revision )
@@ -1,9 +1,9 @@
 #coding:utf-8
 import hashlib
+from flask_login import UserMixin, AnonymousUserMixin
 from . import db, login_manager
 from flask import current_app, request
 from datetime import datetime
-
 from werkzeug.security import generate_password_hash, check_password_hash
 from itsdangerous import TimedJSONWebSignatureSerializer as Serializer
 
\ No newline at end of file
